src/main/app.ts

```diff
@@ -34,7 +34,7 @@ app.use(favicon(path.join(__dirname, '/public/assets/images/favicon.ico')));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
-app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static(path.join(__dirname, 'public'), { maxAge: '60s' }));
```

Comment:
Are you sure this actually works? As far as I know, when a browser encounters two headers that are the same in this case, `max-age` it's the last header that is set that "wins". The line below this is setting `max-age` to `0`. So I just wanted to check this with you.

Also, I don't believe the "s" is valid in `max-age` as it only accepts a [numerical value](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#response_directives). Unless Express removes the "s" when this line is interpreted?

My last point I'd make is: is 60 seconds long enough for a user to get through the entire website from entrance to exit? I'm unsure what the `toffee-recipe-backend` actually is, but if it contains forms that a user needs to complete, it's likely to take them longer than 60 seconds, maybe `600` is more appropriate (10 minutes)?

More than happy to discuss these points as required.